Orange-OpenSource/its-client
# Fix Clippy and other warnings

New features
--------------------

* rust: enable Clippy in Github workflow
* rust: no longer restrict workflows to master branch or pull requests
* rust: fixed several Clippy warnings
* rust: `pipeline::run` method now takes a single `rumqttc::MqttOptions`argument  instead of several arguments

Fixes #68 
Fixes #69 

How to test
-----------------

1. Run the CopyCat example
    ```
    cargo run --bin its-client -- -H <broker_host> -P <broker_port> -u <username> -p <password> --mqtt-client-id <client_id>
    ```
    **=> stdout logs tells that the copycat receives messages (cam, denm, ...)**  

2. Check the applicative logs
    ```
    tail -F log/its-client_rCURRENT.log
    ```
    **=> Logs must tell about either copy/retain moving vehicle messages or stopped vehicle that are not copied**
